src/test/java/kitchenpos/menu/application/MenuGroupServiceTest.java

```diff
@@ -0,0 +1,76 @@
+package kitchenpos.menu.application;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+import java.util.Arrays;
+import java.util.List;
+import kitchenpos.application.MenuGroupService;
+import kitchenpos.dao.MenuGroupDao;
+import kitchenpos.domain.MenuGroup;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@DisplayName("메뉴 그룹 서비스 테스트")
+@ExtendWith({MockitoExtension.class})
+class MenuGroupServiceTest {
+
+    private static final String MENU_GROUP_NAME = "메뉴그룹";
+    private final MenuGroup menuGroup = new MenuGroup();
+
+    @Mock
+    private MenuGroupDao menuGroupDao;
+
+    @InjectMocks
+    private MenuGroupService menuGroupService;
+
+    @BeforeEach
+    void setUp() {
+        menuGroup.setId(1L);
+        menuGroup.setName(MENU_GROUP_NAME);
+    }
+
+    @Test
+    @DisplayName("메뉴 그룹을 등록한다.")
+    void create() {
+        when(menuGroupDao.save(any(MenuGroup.class)))
+            .thenReturn(menuGroup);
+
+        MenuGroup saved = menuGroupService.create(menuGroup);
+
+        assertThat(saved.getName()).isEqualTo(MENU_GROUP_NAME);
+    }
+
+    @Test
+    @DisplayName("메뉴 그룹 목록을 조회한다.")
+    void list() {
+        when(menuGroupDao.findAll())
+            .thenReturn(Arrays.asList(menuGroup));
+
+        List<MenuGroup> menuGroups = menuGroupService.list();
+
+        assertAll(() -> {
+            assertThat(menuGroups.size()).isEqualTo(1);
+            assertThat(menuGroups).extracting(MenuGroup::getName).containsExactly(MENU_GROUP_NAME);
+        });
```

Comment:
```suggestion
        assertAll(
                () -> assertThat(menuGroups.size()).isEqualTo(1), 
                () -> assertThat(menuGroups).extracting(MenuGroup::getName).containsExactly(MENU_GROUP_NAME)
        );
```

assertAll을 사용하는 이유에 대해 고민 후 기존과 같이 작성하면 안되는 이유에 대해 고민해보아요 :)

Reply:
깊게 생각해보지 않았었는데 검증을 독립적으로 하려고 사용했던건데 결국 의존적으로 되어 버렸군요!!!